dspot/src/main/java/eu/stamp_project/dspot/selector/PitMutantScoreSelector.java

```diff
@@ -235,6 +239,14 @@ public TestSelectorElementReport report() {
     }
 
     private String reportStdout() {
+        if (MongodbManager.getInstance().getDbConnectable()) {
```

Comment:
Hi @Tailp,

I would like the following if block in a dedicated method, like `reportMongoDB()`.

Basically, create dedicated methods for all actions that are done on the mongoDB, in order to separate the mongoDB management from the rest of the code.

Reply:
ah I think I understand your point :). I made some changes just now. Hopefully this passes the test on the CI  and is what you wanted.

Reply:
hi @danglotb, I put all codeblocks into reportMongoDB() functions :) . I would be nice if you have some spare time to check if it's ok to go or if some more fixes I need to do. Thank you for your patience.